src/MSBuildSummaryFiles.Tasks/build/MSBuildSummaryFiles.targets

```diff
@@ -57,6 +57,11 @@ Licensed under the MIT license.
       <_SummaryFileTargetFramework Condition="'$(_SummaryFileTargetFramework)' == '' And '$(_TargetFrameworkVersionWithoutV)' != ''">net$(_TargetFrameworkVersionWithoutV.Replace('.', ''))</_SummaryFileTargetFramework>
     </PropertyGroup>
 
+    <ItemGroup>
+      <SourceFiles Include="@(Compile);@(EmbeddedResource)"/>
```

Comment:
A bit worried that SourceFiles is too common a name and might clash with the users' other items. Perhaps make it something more specific such as `MSBuildSummarySourceFiles`?

Reply:
Changed to SummarySourceFiles